systemtest/src/test/resources/oauth2/teardown_keycloak_operator.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+NAMESPACE=$1
+
+KEYCLOAK_VERSION=11.0.0
```

Comment:
I wonder if we need to the parametrized version of the keycloak for the future.

Reply:
I don't think so. We should change version when new version of keycloak will be released and our oauth will work with it. I don't think we care if it's working with 8.0.x , 9.x.y, etc.